<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>153</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>391</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>122</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>122</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>7</y>
     <width>371</width>
     <height>105</height>
    </rect>
   </property>
   <property name="title">
    <string>HTTP Proxy </string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>19</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Proxy Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>57</y>
      <width>97</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Proxy Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>18</y>
      <width>124</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Proxy Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>59</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Proxy Port</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LoginNameEdit">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>34</y>
      <width>169</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordEdit">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>72</y>
      <width>168</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="proxyAddressEdit">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>35</y>
      <width>158</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="proxyPortEdit">
    <property name="geometry">
     <rect>
      <x>199</x>
      <y>73</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="enabledCheckBox">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>13</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Enabled</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>LoginNameEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>proxyAddressEdit</tabstop>
  <tabstop>proxyPortEdit</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>OkButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
